configs/pie-components-config/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@justeattakeaway/pie-components-config",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
I realise the `pie-webc-core` package is also private. Is there any reason we might want to make them both public?

Reply:
Good question. 

I'd say, at least for now, `pie-components-config` is more like internal, dev-oriented content being shared as a workspace, than a package consumers might have interest in.

Do you have any opinions on that? I might be missing something...